pipeline.yml

```diff
@@ -618,49 +626,31 @@ jobs:
               ignorefile: instana-agent-docker-git/.twistlockignore-amd64-static
   - name: verify-amd64-static
     max_in_flight: 1
-    on_abort: &destroy-amd64-static-vm
-      do:
-        - task: destroy-vm
-          file: instana-agent-docker-git/ci/destroy-verification-vm-task.yml
-          vars:
-            instance-name: instana-agent-amd64-static
-            gcp-project: instana-agent-qa
-            gcp-zone: europe-west3-c
-    on_success:  *destroy-amd64-static-vm
-    on_error: *destroy-amd64-static-vm
-    on_failure:  *destroy-amd64-static-vm
     plan:
-      - get: instana-agent-docker-git
-        passed: [ amd64-build ]
-      - get: agent-version
-        passed: [ amd64-build ]
-      - get: agent-static-amd64
-        passed: [amd64-build]
-        trigger: true
-      - load_var: digest
-        file: agent-static-amd64/digest
-      - task: start-vm
-        file: instana-agent-docker-git/ci/create-verification-vm-task.yml
+      - in_parallel:
+        - get: instana-agent-docker-git
+          passed: [ amd64-build ]
+        - get: agent-version
+          passed: [ amd64-build ]
+        - get: agent-static-amd64
+          params:
+            format: oci
+          passed: [amd64-build]
+          trigger: true
+      - put: metadata
```

Comment:
I think the `- put: metadata` step can also be done in the `- in_parallel:` block

Reply:
fixed